enzyme/Enzyme/GradientUtils.cpp

```diff
@@ -3821,6 +3821,11 @@ Value *GradientUtils::invertPointerM(Value *const oval, IRBuilder<> &BuilderM,
       }
     }
 
+    if (auto arg = dyn_cast<ConstantExpr>(oval)) {
+      auto rule = [&oval]() { return Constant::getNullValue(oval->getType()); };
```

Comment:
Oh you don't want to return the null, here but arg (null case is already taken care of above.

Reply:
Thanks, I was already wondering about the test-case changes.